pygmt/tests/baseline/test_basemap.png.dvc

```diff
@@ -1,4 +1,4 @@
 outs:
-- md5: 3cf01816fa5dd3fbc1adc602557bb032
+- md5: 5fc8bd2d28e23b6581733227dc9838a1
```

Comment:
This test isn't actually failing, so maybe we should revert it to keep the diff small? Maybe something to do with differing ghostscript versions or operating systems.

```suggestion
- md5: 3cf01816fa5dd3fbc1adc602557bb032
```

Reply:
I was surprised by this. I accidentally added a PNG from a passing test to DVC and was surprised when the .dvc file changed, so I decided to add all off them. What I don't understand is how my pull requests could produce any baseline images that would pass the CI tests, since there is no "close enough" for a hash.